### PR TITLE
fix: AssertionError: no block info

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

DAPI core smoke tests were failing on all evonodes with:
```
  10) Smoke
       DAPI
         All nodes
           hp-masternode-1
             should return data from Core:
     AssertionError: no block info
      at Context.<anonymous> (test/smoke/dapi.js:92:20)
      at process.processImmediate (node:internal/timers:476:21)
```

And smoke tests were failing with the same issue previously fixed in #550 

## What was done?

Fix test logic

## How Has This Been Tested?

Locally against testnet

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: test/smoke/dapi.js [hunk 1]
```diff
@@ -88,9 +88,12 @@ describe('DAPI', () => {
             expect.fail(null, null, bestBlockHashError[hostName]);
           }
 
-          if (bestBlockHash[hostName]) {
+          if (!bestBlockHash[hostName]) {
             expect.fail('no block info');
           }
+
+          expect(bestBlockHash[hostName]).to.be.a('string');
+          expect(bestBlockHash[hostName]).to.be.not.empty();
```
Comment (shumkov) [inline test/smoke/dapi.js:96]: You don't need this if you have condition above

**Review by shumkov — CHANGES_REQUESTED** [review]

File: test/smoke/dapi.js (hunk 1 above)
Comment (strophy) [reply test/smoke/dapi.js:96]: Fixed

File: test/smoke_web/dapi.js [hunk 2]
```diff
@@ -123,7 +123,8 @@ describe('DAPI', () => {
           }
 
           expect(dataContract[hostName]).to.be.undefined();
-          expect(dataContractError[hostName].code).to.be.equal(5);
```
Comment (shumkov) [inline test/smoke_web/dapi.js:126]: Let's not remove it for now. Just comment out please. We will bring this back at some point

Comment (strophy) [reply test/smoke_web/dapi.js:126]: It was removed in #550 so let's just stay consistent with that?

Comment (shumkov) [reply test/smoke_web/dapi.js:126]: Oh.. that's not great. Could you please recover that one as well and put todos to bring this back?

**Pull request merged**